pytorch_lightning/callbacks/__init__.py

```diff
@@ -48,6 +48,7 @@
     "ProgressBar",
     "ProgressBarBase",
     "QuantizationAwareTraining",
+    "INCQuantization",
```

Comment:
Let's keep this list alphabetically ordered.

Reply:
Done